src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
Enum values must be in uppercase
```suggestion
    BLACK,
```